version.c

```diff
@@ -200,6 +195,8 @@ rb_ruby_default_parser_set(ruby_default_parser_enum parser)
 static void
 define_ruby_description(const char *const jit_opt)
 {
+#define JIT_DESCRIPTION YJIT_DESCRIPTION ZJIT_DESCRIPTION
```

Comment:
I was wondering if we could reduce the number of wasted bytes in the buffer, but even if we somehow pick the longer description here at preprocessor time, when a JIT with a shorter description is enabled, we'd have to waste some bytes anyway. So I guess this is fine.